src/compress.py

```diff
@@ -0,0 +1,48 @@
+class CompressString:
+	def __init__(self, string='', letter=None, compression='', count=0):
+		self.string = string
+		self.compression = compression
+		self.letter = letter
+		self.count = count
```

Comment:
To avoid introducing subtle bugs, a best practice is to use either a specific value or `None` as the default for arguments.

There's this idea of "defensive" code that we want to avoid completely*. It's any code that tries to guess what the input might be. If you're working in an area where you determine the inputs and outputs, just be consistent. 

\* _with a couple of exceptions, like in places where you don't have control over the input values or when it's acceptable to continue ignoring exceptions._